src/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   getWilayaByZipCode: getWilayaByZipCode(_getData()),
   getZipCodesForWilaya: getZipCodesForWilaya(_getData()),
   getDairatsForWilaya: getDairatsForWilaya(_getData()),
+  getWilayaByPhoneCode: getWilayaByPhoneCode(_getData()),
```

Comment:
Please update `index.test.js` with this change

Reply:
[FIXED]